docs/Readme.md

```diff
@@ -1,24 +1,3 @@
 # Building ipywidgets documentation
 
-To build the ipywidgets documentation:
-
-1. Install the development version of ipywidgets with the `dev-install.sh` script in the repo root directory.
-2. Install the Python packages for building documentation with either mamba or conda:
-   ```sh
-   conda env update --file docs/environment.yml 
-   ```
-
-   or with `pip`:
-
-   ```sh
-   python -m pip install -r docs/requirements.txt
-   # Also install pandoc separately
-   ```
-
-3. Build the HTML docs with sphinx:
-   ```sh
-   cd docs/source
-   python -m sphinx -T -E -b html -d ../build/doctrees -D language=en . ../build/html
-   ```
-
-4. Open the documentation from the build directory: `docs/build/html`
+To build the ipywidgets documentation see the instructions available on - [Jupyter Widgets - Develop and Build Documentation](https://ipywidgets.readthedocs.io/en/latest/dev_install.html)
```

Comment:
We may want to link the following instead: https://ipywidgets.readthedocs.io/en/latest/dev_docs.html

Reply:
Or rather, this seems to the correct link we want to provide: https://github.com/jupyter-widgets/ipywidgets/tree/master/docs

Reply:
(Since the changes to the dev_docs.md seem to be mirroring that^)

Reply:
Instructions in the readme were removed so that we would not have to maintain two different sets of the same instructions on both the readme and the ipywidgets.readthedocs.io, but i can add back the readme instructions as well

Reply:
It makes sense to maintain one set of instructions however it seems that the instructions here https://ipywidgets.readthedocs.io/en/latest/dev_install.html is for the dev install of ipywidgets, not specifically for documentation. And if we were to link https://ipywidgets.readthedocs.io/en/latest/dev_docs.html instead, this seems to be different set of instructions from the README instructions that you updated